private/bufpkg/bufplugin/bufplugin.go

```diff
@@ -239,10 +244,59 @@ func ProtoRegistryConfigToPluginRegistry(config *registryv1alpha1.RegistryConfig
 			return nil, err
 		}
 		registryConfig.Maven = mavenConfig
+	} else if protoSwiftConfig := config.GetSwiftConfig(); protoSwiftConfig != nil {
+		swiftConfig, err := ProtoSwiftConfigToSwiftRegistryConfig(protoSwiftConfig)
+		if err != nil {
+			return nil, err
+		}
+		registryConfig.Swift = swiftConfig
 	}
 	return registryConfig, nil
 }
 
+func ProtoSwiftConfigToSwiftRegistryConfig(protoSwiftConfig *registryv1alpha1.SwiftConfig) (*bufpluginconfig.SwiftRegistryConfig, error) {
+	swiftConfig := &bufpluginconfig.SwiftRegistryConfig{}
+	runtimeLibs := protoSwiftConfig.GetRuntimeLibraries()
+	if runtimeLibs != nil {
+		swiftConfig.Deps = make([]bufpluginconfig.SwiftRegistryDependencyConfig, 0, len(runtimeLibs))
+		for _, runtimeLib := range runtimeLibs {
+			swiftConfig.Deps = append(swiftConfig.Deps, bufpluginconfig.SwiftRegistryDependencyConfig{
+				Module:   runtimeLib.GetPackage(),
```

Comment:
It seems like these fields should line up, i.e. either it should be `package` or `module`, not both?